tests/system/common.py

```diff
@@ -57,8 +57,9 @@ def job_with_secrets(id='pikachu',
 
 def job_with_file_based_secret(
         id='pikachu-fbs',
-        cmd='cat $MESOS_SANDBOX/secret-file > $MESOS_SANDBOX/fbs-secret; sleep 5',
+        cmd='cat $MESOS_SANDBOX/secret-file > $MESOS_SANDBOX/fbs-secret; sleep 30',
```

Comment:
the sleep time is not important... the job and all running tasks will be destroyed as soon as the test passes.